2/answers.txt

```diff
@@ -0,0 +1,231 @@
+Q1: Why is password authentication inferior to public key authentication?
+
+Because Key based authentication is computer generated, rather than human generated, and typically 4096 Bits in length
+less prone to being brute forced, acquired by subterfuge or guessed than a traditional password. 
+
+Communications between two system using a key pair method is only possible to decrypted by owners of the key pair.
+Furthermore a private key can typically be further encrypted with a standard password to increase safety.
```

Comment:
Password authentication involves transmitting the username and password in some way.
Usually this is a non-issue with an encrypted communication channel (eg HTTPS). However, it's still possible that the secure data will end up in a log in plain text at some point, which is another potential security threat. 

When using a key, these issues are avoided as all data sent is encrypted with the key.